modules/govuk_jenkins/templates/jobs/copy_data_from_integration_to_aws.yaml.erb

```diff
@@ -39,8 +39,6 @@
       - env-sync-and-backup_Copy_Data_from_Integration_to_AWS
     logrotate:
         numToKeep: 10
-    triggers:
-        - timed: 'H 11 * * 1-5'
```

Comment:
Why is this being removed?

Reply:
The job takes more than 24 hours to complete. Also, it only needs to run when we need.

Reply:
Why change this? We only have one integration environment now, and that is running in AWS. Do you have a plan to use this job?

Reply:
Hi @cbaines ,

I will answer the questions below. Please let me know if further details are needed.

Why change this?
When this job actually ran, it took a significant time and we only need this job to run when required [These are related to Carrenza staging]. We removed this to match the job in Staging and Production.

Do you have a plan to use this job?
This job is not currently deployed. The change was made to match the changes on other environments https://github.com/alphagov/govuk-puppet/blob/master/hieradata/class/integration/jenkins.yaml.

Reply:
>>Why change this?

> When this job actually ran, it took a significant time and we only need this job to run when required [These are related to Carrenza staging]. We removed this to match the job in Staging and Production.

>> Do you have a plan to use this job?

> This job is not currently deployed. The change was made to match the changes on other environments https://github.com/alphagov/govuk-puppet/blob/master/hieradata/class/integration/jenkins.yaml.

It sounds like you don't plan on using it in the future? If this is the case, I'd suggest removing it, rather than trying to keep it similar to the jobs for staging and production. Save yourself the effort, and make the history of the project neater for the future.

Reply:
https://trello.com/c/B2A7UkrQ/63-remove-obsolete-data-sync

Reply:
Job added to the to-do-list